src/Knp/Menu/MenuItem.php

```diff
@@ -263,7 +263,7 @@ public function getChildrenAttribute(string $name, $default = null)
         return $default;
     }
 
-    public function setChildrenAttribute(string $name, $value): ItemInterface
+    public function setChildrenAttribute(string $name, string $value): ItemInterface
```

Comment:
It looks like casting `string` on attribute's value is too strong.
Indeed, the old code (see branch v2) suggests `string` in phpdoc, but likely the attribute's value could be something else.
So, please remove casting from class **and** from interface

Reply:
Changes made although I see `setLinkAttribute(string $name, string $value)` still has `$value` casted as a `string`.

Reply:
You're right, it should be removed as well.

Reply:
See PR #314